client/src/annoMatrix/annoMatrix.ts

```diff
@@ -227,6 +237,36 @@ export default abstract class AnnoMatrix {
     return annoMatrix;
   }
 
+  abstract addObsColumn(
```

Comment:
New abstract methods expand the AnnoMatrix contract (addObsColumn, addObsAnnoCategory, drop/rename/set/reset values). Please sanity‑check naming, invariants, and compatibility with existing crossfilter/loader usage.